src/main/java/edu/berkeley/cs186/database/query/SortOperator.java

```diff
@@ -86,8 +86,17 @@ public Iterator<Record> iterator() {
      * iterator
      */
     public Run sortRun(Iterator<Record> records) {
-        // TODO(proj3_part1): implement
-        return null;
+        // TODO(proj3_part1): implement Done
+        // Create a new run
+        Run newRun = makeRun();
+        List<Record> inputRecords = new ArrayList<>();
+        while (records.hasNext()) {
+            inputRecords.add(records.next());
+        }
+        inputRecords.sort(comparator);
+        newRun.addAll(inputRecords);
```

Comment:
```suggestion
        return makeRun(inputRecords);
```
List<Record>을 매개변수로 받아 Run 객체를 만들어주는 메소드도 아래 구현되어 있습니다..!